variables.tf

```
@@ -40,16 +40,16 @@ variable "versioning_enabled" {
   default     = true
 }
 
-variable "noncurrent_version_expiration_days" {
-  type        = number
-  default     = 90
-  description = "Specifies when noncurrent object versions expire"
+variable "enable_noncurrent_version_expiration" {
```

We normally use this convention `<NAME>_enabled` for new booleans, no ?

```suggestion
variable "noncurrent_version_expiration_enabled" {
```

---

Oh I see, this is a pass-thru input to the s3-bucket module. This is going against our convention. I'd rather persist our new convention and correct the upstream s3-bucket module.

---

Yes, but this release (the next one, too) has enough issues with moving resources, I do not want to introduce breaking changes to the interface.